src/pytest_forked/__init__.py

```diff
@@ -71,13 +71,8 @@ def runforked():
 
 
 def report_process_crash(item, result):
-    try:
-        from _pytest.compat import getfslineno
-    except ImportError:
-        # pytest<4.2
-        path, lineno = item._getfslineno()
-    else:
-        path, lineno = getfslineno(item)
+    from _pytest._code.source import getfslineno
+    path, lineno = getfslineno(item)
```

Comment:
FTR this returns `path = ''` and `lineno = -1` in my setup. Dunno if it's generally broken in pytest 5.4. It may need some extra investigation.